<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Блокнот</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Язык</string>
     </property>
     <addaction name="russian"/>
     <addaction name="english"/>
    </widget>
    <addaction name="open_2"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="separator"/>
    <addaction name="keysetting"/>
    <addaction name="menu_2"/>
    <addaction name="help"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="open_2">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="openRead">
   <property name="text">
    <string>Открыть в режиме чтения</string>
   </property>
  </action>
  <action name="russian">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="english">
   <property name="text">
    <string>Английский</string>
   </property>
  </action>
  <action name="keysetting">
   <property name="text">
    <string>Настройки раскладки клавиатуры</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
